Source/SvgElementStyle.cs

```diff
@@ -154,7 +154,7 @@ public virtual float Opacity
         [SvgAttribute("shape-rendering")]
         public virtual SvgShapeRendering ShapeRendering
         {
-            get { return GetAttribute<SvgShapeRendering>("shape-rendering", true); }
+            get { return GetAttribute<SvgShapeRendering>("shape-rendering", true, SvgShapeRendering.Auto); }
```

Comment:
can omit type `<SvgShapeRendering>` in method.

Reply:
The type was in original, I can remove it if that's requested.

Reply:
If `defaultValue` is specified, type is omitted for other properties.

Reply:
I will remove the unnecessary type.

Reply:
Pushed the change.